tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -54,6 +54,15 @@ public async Task Interactive_AADAsync()
             await RunTestForUserAsync(labResponse).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task Interactive_MsaUser_Async()

```

Comment:
We should also have an ROPC test with MSA, where we should assert that it fails gracefully.